docs/platforms/javascript/common/configuration/integrations/openfeature.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: OpenFeature
+description: "Learn how to use Sentry with OpenFeature."
+notSupported:
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.bun
+  - javascript.capacitor
+  - javascript.cloudflare
+  - javascript.connect
+  - javascript.cordova
+  - javascript.deno
+  - javascript.electron
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.nodejs
+  - javascript.wasm
+---
+
+<PlatformContent includePath="feature-flags/prerelease-alert" />
+
+<Alert level="info">
+
+This integration only works inside a browser environment.
+
+</Alert>
+
+The [OpenFeature](https://openfeature.dev/) integration tracks feature flag evaluations produced by the OpenFeature SDK. These evaluations are held in memory, and in the event an error occurs, sent to Sentry for review and analysis. **At the moment, we only support boolean flag evaluations.**
+
+_Import name: `Sentry.openFeatureIntegration` and `Sentry.OpenFeatureIntegrationHook`_
+
+## Install
+
+Install [`@sentry/browser`](https://www.npmjs.com/package/@sentry/browser) and [`@openfeature/web-sdk`](https://www.npmjs.com/package/@openfeature/web-sdk) from npm.
+
+## Configure
+
+```JavaScript
+import * as Sentry from '@sentry/browser';
+import { OpenFeature } from '@openfeature/web-sdk';
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [Sentry.openFeatureIntegration()]
+});
+
+OpenFeature.setProvider(new MyProviderOfChoice());
+
+// Option 1: track all OpenFeature evaluations.
+OpenFeature.addHooks(new Sentry.OpenFeatureIntegrationHook());
+
+// Option 2: only track evaluations by a specific client.
+const client = OpenFeature.getClient();
+client.addHooks(new Sentry.OpenFeatureIntegrationHook());
+```
+
+Learn more about OpenFeature providers [here](https://openfeature.dev/docs/reference/concepts/provider).
+
+## Verify
+
+The integration is tested by evaluating a feature flag with your OpenFeature SDK before capturing an exception.
+
+```JavaScript
+import * as Sentry from '@sentry/browser';
+import { OpenFeature } from '@openfeature/web-sdk';
+
+// Evaluate a flag with a default value. If you added the hook to a client in
+// the Configure step, make sure to use the same client here.
+const client = OpenFeature.getClient();
+const result = client.getBooleanValue('hello', false);
+
+Sentry.captureException(Exception("Something went wrong!"))
+```
+
+Visit the Sentry website and confirm that your error event has recorded the feature flag "hello" and its value "false". Learn more about the OpenFeature SDK at https://github.com/open-feature/js-sdk/tree/main.
```

Comment:
there's no equiv of the "Learn more about the OpenFeature SDK at ___" sentence on the python page